utils.go

```diff
@@ -7,6 +7,8 @@ package fiber
 import (
 	"bytes"
 	"fmt"
+	"github.com/valyala/bytebufferpool"
```

Comment:
The third part libraries should be imported after standard library, how about changing to:

```
"hash/crc32"
"net"		"net"
"strings"		"strings"

"github.com/valyala/bytebufferpool"
"github.com/valyala/fasthttp"
```

Reply:
Fixed. Thanks to point this out.